devito/ops/operator.py

```diff
@@ -11,6 +13,10 @@ class OperatorOPS(Operator):
     """
 
     def _specialize_iet(self, iet, **kwargs):
+        for n, (section, trees) in enumerate(find_affine_trees(iet).items()):
+            callable_kernel, par_loop_call_block = opsit(trees, n)
```

Comment:
As you are not using the `par_loop_call_block` variable yet, I think you could just drop it for now:

`callable_kernel, _ = opsit(trees, n)`